Training/etc/di.xml

```diff
@@ -22,5 +22,5 @@
         </arguments>
     </type>
     <preference for="Codifi\Training\Api\NoteRepositoryInterface" type="Codifi\Training\Model\NoteRepository"/>
-    <preference for="Codifi\Training\Api\Data\NoteInterface" type="Codifi\Training\Model\Note"/>
+    <preference for="Codifi\Training\Api\Data\NoteInterface" type="Codifi\Training\Model\CustomerNote"/>
```

Comment:
If you rename class you have to update all dependencies in all classes where you inject this model (factory).